modules/plugin/geopkg/src/test/java/org/geotools/geopkg/GeoPackageTest.java

```diff
@@ -1123,4 +1120,82 @@ private void createBugSites() throws Exception {
         geopkg.add(entry, shp.getFeatureSource(), null);
         assertTableExists("bugsites");
     }
+
+    // if the FC already is XY, then forceXY does nothing (should return same FC)
+    @Test
+    public void testForceXY_alreadyXY() throws Exception {
```

Comment:
Unusual name (no java camelCase), I can see what you are trying to do. Is their a JUnit convention to follow?

Reply:
I can change this - but is common to see "_" in test case names.  I wanted to make the function name obvious as to what its doing.  I will change to `testForceXYAlreadyXY` and the same, below.